### ISSUE TITLE
Controls to like and repost

### ISSUE THREAD

**Issue by maertz** [salomvary/soundcleod#143]
Hey @salomvary, it would be pretty handy if the controls work as well while your app is hidden (well media keys work).
In addition it would be awesome if you add the possibility to like and repost the current track
– Ben

**Comment by salomvary** [comment]
> it would be pretty handy if the controls work as well while your app is hidden

Yes, that would be nice (although only works on macOS where an application can be activated with no visible windows on the screen).

Unless you mean *global* shortuct which could work on all operationg systems and no matter what application is visible or active. That's a little problematic because it would be really hard to come up with a global shortcut that does not potentially conflict with some other application using the same shortcut. Even if we decide to go with global shortcuts that will then need to be configurable, making it more complicated to build. Anyway, I'm not saying it's impossible, it's just quite a bit of work.

> Besides that it would be awesome if you could add the possibility to like and repost the current track.

That already works, when SoundCleod is the active window on the screen, just hit "r". Also, hit "h" to see more keyboard shortcuts.

**Comment by ErraticFox** [comment]
The global feature wouldn't really tend to me. Though for others that would like it, I'd have an idea for implementation. As this would be a case-by-case, you could have an option you could enable (default: disabled) where they could bind any key-combo to do a `repost` or `like`. As well as adding a little side-note saying that it could have potential issues with other programs.
